src/page-modules/assistant/trip/trip-pattern/index.tsx

```diff
@@ -20,6 +20,16 @@ type TripPatternProps = {
   index: number;
 };
 
+const isEqualOnMinuteLevel = (
```

Comment:
Maybe we could use the `secondsBetween` function that already exists in `src/utils/date.ts` ?

Reply:
I´ll check it out👍